README.md

```diff
@@ -157,20 +167,20 @@ the chosen segmentation service, and pass the video along.
 def post_video_content(host, port, camera_name, camera_id, filepath, timestamp, latlng=None):
     """
     arguments:
-        host        - the url of the segmenter # TODO(carter) call this url or server since host is confusing.
+        host        - the url of the segmenter
         port        - the port to access the webserver on the segmenter
         camera_name - the parsed name of the camera
-        camera_id   - the ID of the camera as returned from the register_camera function
-        filepath    - full path to the video file that needs segmentation
-        timestamp   - the earliest timestamp contained in the video file
-        location (opt) - a string describing the location of the camera. 
-                         example: {"location": {"lat": 7.367598, "lng":134.706975}, "accuracy":5.0}
+        camera_id   - the ID of the camera as returned from the service
+        filepath    - full path to the video file that needs to be uploaded for segmentation
+        timestamp   - the starting timestamp of the video file
+        location(opt) - a json-string describing the location of the camera
```

Comment:
This might be misinterpreted as a "JSON string" literally. So let's say:
"String of JSON describing..."